src/ECS/Components/StoragePit.h

```diff
@@ -0,0 +1,25 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <array>
+
+#include <entt/fwd.hpp>
+
+namespace openblack::ecs::components
+{
+
+struct StoragePit
+{
+	std::array<entt::entity, 5> woodPiles;
```

Comment:
Where did the 5 come from?

Reply:
There are 5 separate piles here
<img width="354" alt="image" src="https://user-images.githubusercontent.com/1013356/184540750-e6c3076b-c25f-41f0-9089-59e2dd5728ec.png">

Reply:
Ah ok got it